packages/apps/dms/src/hooks/use-dms-header-actions.ts

```diff
@@ -49,3 +52,25 @@ const useAllDocumentsActions = () => {
     });
   }, [Colors, I18n, navigation]);
 };
+
+const useAttachedFilesGenericAction = () => {
+  const I18n = useTranslator();
+  const navigation = useNavigation();
+
+  const {mobileSettings} = useSelector((state: any) => state.appConfig);
+
+  useEffect(() => {
+    headerActionsProvider.registerGenericAction(
+      'attached_files_generic_action',
```

Comment:
Follow a similar convention as normal header actions
Here something like dms_attachedFiles as it's coming from the DMS package
Not sure adding the generic action is necessary as you are using registerGenericAction
We need to try to make those keys as simple as possible as it will be used for the options